### ISSUE TITLE
[BUG] "Run at Startup" set wrong path in Windows Registy

### ISSUE THREAD

**Issue by fubaWoW** [radj307/volume-control#125]
### Version Number

6.5.2

### File

Portable   (VolumeControl.exe)

### Windows Version

Windows 11 22H2 Build 22624.2428

### Actual Behaviour

The ``Run at Startup`` set a wrong or partially wrong path in the Windows Registry.

Actual Windows Registry if ```Run at Startup`` is enabled
```reg
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run]
"VolumeControl"="D:\\Settings\\Tools\\VolumeControl
```

Screenshot:
![image](https://github.com/radj307/volume-control/assets/41425685/1255f6d8-5e59-4a9b-ae35-c97cfa053bef)

### Expected Behaviour

But this Registry key should be set

```reg
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run]
"VolumeControl"="D:\\Settings\\Tools\\VolumeControl\\VolumeControl.exe"
```

Screenshot:
![image](https://github.com/radj307/volume-control/assets/41425685/6520fc9c-514a-4693-8401-86faafaf5247)

also you should set this

```reg
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run]
"VolumeControl"=hex:02,00,00,00,00,00,00,00,00,00,00,00
```

Screenshot:
![image](https://github.com/radj307/volume-control/assets/41425685/ffbedb7f-005c-4d1b-8f1d-64f23c46a84a)

### Log

Log file does not help (again), but here:
```
HH:mm:ss:fff              === Log Initialized @ Sonntag, 12. November 2023 20:13:15 ===  { LogFilter: 0 (NONE) }
21:13:29:545  [TRACE]     Config property 'LogFilter' was modified.
21:13:29:545  [TRACE]     Saved Config (Caller: "HandlePropertyChanged")
21:13:32:748  [TRACE]     Config property 'RunAtStartup' was modified.
21:13:32:748  [TRACE]     Saved Config (Caller: "HandlePropertyChanged")
21:13:33:033  [TRACE]     Config property 'RunAtStartup' was modified.
21:13:33:033  [TRACE]     Saved Config (Caller: "HandlePropertyChanged")
21:14:32:936  [TRACE]     Config property 'RunAtStartup' was modified.
21:14:32:936  [TRACE]     Saved Config (Caller: "HandlePropertyChanged")
21:14:33:468  [TRACE]     Config property 'RunAtStartup' was modified.
21:14:33:468  [TRACE]     Saved Config (Caller: "HandlePropertyChanged")

```

**Comment by radj307** [comment]
On it.

**Comment by radj307** [comment]
Regarding setting a value in `HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run`, it probably isn't necessary. It's used by the Task Manager to enable/disable run at startup entries, and isn't generally managed by applications.  
If Volume Control needed to *force* itself to run at startup regardless of whether the user disabled it through the task manager, that's the key to use.

Either way, I'll build the functionality to do this in, but the actual implementation probably won't use it.

**Comment by radj307** [comment]
Should be fixed in 5ed767117177d8008382f10721000ff68ac395a1

A test build is being built now.

**Comment by radj307** [comment]
[Here is a test build](https://github.com/radj307/volume-control/actions/runs/6843409224) with the fix, and [download instructions](https://github.com/radj307/volume-control/wiki/Downloading-Action-Artifacts) should you need them.

**Comment by fubaWoW** [comment]
The Path is now correct and works but (for me) only if ``StartupApproved`` is set.

```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run]
"VolumeControl"=hex:02,00,00,00,00,00,00,00,00,00,00,00
```

As you can see in the screenshot below, many 3rd party apps added themself to ``StartupApproved``, so this is maybe needed in Windows 11?

![image](https://github.com/radj307/volume-control/assets/41425685/f2217965-44ff-48fe-b58a-6350dafc417b)

**Comment by radj307** [comment]
Usually the Task Manager adds that on its own *(`StartupApproved` was set automatically in my case)*, but if that didn't work on your system then there is a good argument for setting that explicitly.

**Comment by radj307** [comment]
As of f9ad6a8605077463a318db1927223abad21fe673 a value in `StartupApproved\Run` is created explicitly.

**Comment by radj307** [comment]
Now available in [v6.6.1](https://github.com/radj307/volume-control/releases/tag/6.6.1)

**Comment by fubaWoW** [comment]
Thank you, i will test it later 😎
